multicluster-hub-operator/subscription.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: multicluster-hub-operator-subscription
+spec:
+  channel: integration-2.0
```

Comment:
Since we are not yet really doing stage-related channels, I decided that this channel name was a bad idea for now.  So it will be `snapshot-2.0` in the bundle.  Can you update this subscription resource accordingly?